src/pages/search.vue

```diff
@@ -56,11 +54,25 @@ const BrowsePage = {
     const isMinScreenMd = isMinScreen('md')
     const { isVisible } = useFilterSidebarVisibility()
     const showScrollButton = inject('showScrollButton')
+    const searchStore = useSearchStore()
+
+    const query = computed(() => searchStore.query)
+    const searchType = computed(() => searchStore.searchType)
+    const searchQueryParams = computed(() => searchStore.searchQueryParams)
+    const supported = computed(() =>
+      supportedSearchTypes.includes(searchType.value)
+    )
```

Comment:
This `supported` value appears to be calculated redundantly in a few places. Should we create a getter for it instead? `searchStore.isSelectedSearchTypeSupported`?

Reply:
I removed the `supported` computed from VSearchGrid with the prop that is passed anyways, and also extracted the `isValidSearchType` prop validator. So, now `supported` value is only calculated once (except for prop validation), and there is probably no need for a getter.